tests/gradients/test_parameter_shift.py

```diff
@@ -324,6 +333,41 @@ def test_f0_provided(self):
 
         fn(dev.batch_execute(tapes))
 
+    def test_op_with_custom_unshifted_term(self):
+        """Test that an operation with a gradient recipe that depends on
+        its instantiated parameter values works correctly within the parameter
+        shift rule. Also tests that grad_recipes supersedes paramter_frequencies.
+        """
+
+        class RX(qml.RX):
+            """RX operation with an additional (wrong) term in the grad recipe."""

```

Comment:
What does wrong refer to here? Analytically or structurally incorrect?

Reply:
Ah, good question! Analytically wrong, I'll clarify that